packages/react-components/react-aria/src/activedescendant/useActiveDescendant.ts

```diff
@@ -143,6 +145,18 @@ export function useActiveDescendant<TActiveParentElement extends HTMLElement, TL
         }
       },
 
+      focusLastActive: () => {
```

Comment:
Please add a test for this - if it ever breaks, without a test it would be quite hard to trace it back

Reply:
Added unit test to useActuveDescendant test class :)